tools/resource_map_creator.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
```

Comment:
Would recommend creating a description of what this file does.

Reply:
done